autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/AutoscaleStackType.java

```diff
@@ -0,0 +1,5 @@
+package com.sequenceiq.periscope.api.model;
+
+public enum AutoscaleStackType {
+    WORKLOAD
```

Comment:
What is the significance of this?, and potential alternate values?

Reply:
There are DATALAKE type clusters also.